collection-renderer-lib/src/cmr/collection_renderer/services/collection_renderer.clj

```diff
@@ -24,11 +26,13 @@
   "The main ERB used to generate the Collection HTML."
   (io/resource "collection_preview/collection_preview.erb"))
 
-(def preview-gem-umm-schema-version
-  "Temporay def for the UMM schema version that is supported by the CMR preview gem.
-   This will be removed once CMR preview gem is fixed to include the UMM schema version
-   it supports in the gem. (MMT-921)"
-  "1.6")
+(def preview-gem-umm-version-config-file
+  "Defines the path to the UMM schema version config file within the preview gem."
+  "gems/cmr_metadata_preview-0.0.1/.umm-version")
```

Comment:
We've not committed to any changes... the requirements and expectations for this gem have gone far beyond what was originally discussed. Before anymore tickets or requests are made we need to get on the same page. MMT uses commit hashes for 'versioning' but we just use the most recent so it was never a concern. I'm not even convinced this should be a ruby gem anymore given the way its being used by CMR...